packages/angular/projects/ui-angular/src/lib/primitives/password-field/password-field.component.ts

```diff
@@ -16,6 +16,7 @@ export class PasswordFieldComponent {
   @Input() placeholder = '';
   @Input() required = true;
   @Input() labelHidden = false;
+  @Output() onBlur = new EventEmitter<Event>();
```

Comment:
This could have been an `Input()` or `Output()`. I felt Output worked here so I left it